iphone/Classes/TiMediaVideoPlayerProxy.m

```diff
@@ -685,7 +683,11 @@ - (void)setSourceType:(id)type
 
 - (NSNumber *)playbackState
 {
-  return NUMINTEGER(_playbackState);
+  if (_playbackState != nil) {
```

Comment:
Can we compare NSInteger  to nil ? And default value of _playbackState will be TiVideoPlayerPlaybackStateStopped.

Reply:
I've commited in the wrong state. Actually, I was thinking of doing an object that could be nil but NSInteger obviously is a primitive type. The solution I came up with is to have `TiVideoPlayerPlaybackStateUnknown` which equals -1 and a) is a good default state and b) enumerates fine to 0 (stopped), 1 (playing) etc.